provider/pkg/quicksight/shared-types.json

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
These are confirmed sharing roots - really would like to keep this work from this PR. It took a bit to confirm and name each one.